internal/transformer/cicdtransformer.go

```diff
@@ -31,57 +31,74 @@ import (
 
 // CICDTransformer creates the necessary tekton artifacts needed for CI/CD.
 type CICDTransformer struct {
-	transformedObjects     []runtime.Object
-	extraFiles             map[string]string // file path: file contents
-	TargetClusterSpec      collecttypes.ClusterMetadataSpec
-	IgnoreUnsupportedKinds bool
+	transformedTektonObjects      []runtime.Object
+	transformedBuildConfigObjects []runtime.Object
```

Comment:
Do we need this wrapper cicdtransformer.go? Would it be possible to flatten it, and have buildconfigtransformer and tektontransformer at base level itself?

Reply:
done